documents/presentations/201809_nkg_poster/demo.tex

```diff
@@ -0,0 +1,27 @@
+Here are some examples of how to use Posetta as a command line program:
+
+\textbf{Get help and information about the program}
+
+\begin{verbatim}
+  $ posetta -h
+\end{verbatim}
+
+\textbf{Convert file from XYZ format to CSV format}
+
+Options \texttt{-f}, \texttt{-F} are used to specify which file and format to convert from. Options \texttt{-t}, \texttt{-T} specify which file to convert to.
+
+\begin{verbatim}
+  $ posetta -f example.xyz -F xyz -t example.csv -T csv
+\end{verbatim}
+
+\textbf{Combine with Proj to transform coordinates in a SOSI file}
```

Comment:
Also, what is a SOSI file? Do we support that at the moment? If not, I think the example should be using a format we do know how to read.

Reply:
I wanted an example with reading and writing to the same format, and at the moment we don't have any formats we support like that :blush: 

But I guess we could quite quickly add readers for CSV and PROJ and make this into a working example (with CSV instead of SOSI)